src/pages/resultsView/enrichments/AlterationEnrichmentsContainer.tsx

```diff
@@ -395,10 +399,65 @@ export default class AlterationEnrichmentContainer extends React.Component<
         );
     }
 
+    @computed private get geneBarplotYAxislabel() {
+        if (
+            this.isAnyMutationTypeSelected &&
+            !this.isAnyFusionTypeSelected &&
+            !this.isAnyCnaTypeSelected
+        )
+            return 'Mutation frequency';
+        if (
+            !this.isAnyMutationTypeSelected &&
+            this.isAnyFusionTypeSelected &&
+            this.isAnyCnaTypeSelected
+        )
+            return 'Fusion event frequency';
+        if (
+            !this.isAnyMutationTypeSelected &&
+            !this.isAnyFusionTypeSelected &&
+            !this.isAnyCnaTypeSelected
+        )
+            return 'Copy-number alteration frequency';
+        return 'Alteration event frequency';
+    }
+
+    @computed get isAnyMutationTypeSelected() {
+        if (!this.props.comparisonStore) return false;
+        const typeMap = this.props.comparisonStore!
+            .selectedMutationEnrichmentEventTypes;
+        return _.find(
+            _.keys(typeMap),
+            type => typeMap[type as MutationEnrichmentEventType]
+        );
+    }
+
+    @computed get isAnyFusionTypeSelected() {
+        if (!this.props.comparisonStore) return false;
+        return !!this.props.comparisonStore!
```

Comment:
you can use in operator to be more precise and clear
``` MutationEnrichmentEventType.fusion in someojbect```

Reply:
Ok, it now is:

```
    @computed get isAnyFusionTypeSelected() {
        if (this.props.comparisonStore
            || MutationEnrichmentEventType.fusion in this.props.comparisonStore!)
            return !!this.props.comparisonStore!
                .selectedMutationEnrichmentEventTypes[
                    MutationEnrichmentEventType.fusion
                ]
        return false;
    }
```